subsystems/chassis/Chassis.java

```diff
@@ -1,51 +1,44 @@
 package org.usfirst.frc4904.standard.subsystems.chassis;
 
-
-import org.usfirst.frc4904.standard.commands.chassis.ChassisIdle;
 import org.usfirst.frc4904.standard.subsystems.motor.Motor;
-import edu.wpi.first.wpilibj.command.Subsystem;
+import edu.wpi.first.wpilibj2.command.Subsystem;
 
 /**
- * Generic Chassis class.
- * In the 4904 standard, the Chassis is treated as a container for motors and as a calculator for motor speeds.
- *
+ * Generic Chassis class. In the 4904 standard, the Chassis is treated as a
+ * container for motors and as a calculator for motor speeds. setDefaultCommand
+ * should be called from RobotMap
  */
-public abstract class Chassis extends Subsystem {
+
+public abstract class Chassis implements Subsystem {
 	protected double[] motorSpeeds;
 	protected Motor[] motors;
 
 	/**
 	 *
-	 * @param name
-	 * @param motors
-	 *        all the motors that are part of this Chassis. Pass from front to back, left to right
+	 * ] * @param motors all the motors that are part of this Chassis. Pass from
+	 * front to back, left to right
 	 */
-	public Chassis(String name, Motor... motors) {
-		super(name);
+	public Chassis(Motor... motors) {
+		// super(name);
 		this.motors = motors;
 		motorSpeeds = new double[motors.length];
 	}
 
-	@Override
```

Comment:
Nevermind. Override the `getDefaultCommand()` method like is done in `SolenoidShifters.java` in this PR